### PR TITLE
Add corsair-icue 3.12.118

### PR DESCRIPTION
**Pull request by ytbmulder**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

The Corsair iCUE software has been available on Windows for some time. The forum post listed as the `appcast` is from a Corsair employee providing a download link to a beta version of the iCUE software.

As far as I know, there is no beta repository for `cask-drivers`. Therefore I am opening a pull request in this repository.

### CODE REVIEW

**Comment by ytbmulder** [comment]
@claui @reitermarkus @vitorgalvao

Locally, `brew cask audit` passes and I can also access the `url` from a browser. 
What could be wrong?
```
audit for corsair-icue-beta: failed
 - The URL https://www.corsair.com/us/en/icue is not reachable (HTTP status code 403)
```

File: Casks/corsair-icue-beta.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+cask 'corsair-icue-beta' do
```
Comment (vitorgalvao) [inline Casks/corsair-icue-beta.rb]: There’s no non-beta cask here. Is there no stable version? If so, [you can remove the `-beta`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Comment (ytbmulder) [reply Casks/corsair-icue-beta.rb]: There is no stable version yet. I made the changes as you requested.

**Comment by vitorgalvao** [comment]
> What could be wrong?

The user-agent we use for it, or something in their server.

I’m not being able to replicate the error, though (but I do see it with `audit`). This one is safe to ignore; checking for page availability is always finicky and depends on factors we sometimes can’t control (or that would require an unreasonable amount of work to go around).

**Comment by ytbmulder** [comment]
> I’m not being able to replicate the error, though (but I do see it with `audit`). This one is safe to ignore; checking for page availability is always finicky and depends on factors we sometimes can’t control (or that would require an unreasonable amount of work to go around).

Thanks for the explanation @vitorgalvao :)

**Pull request merged**